src/Tree.tsx

```diff
@@ -430,6 +437,11 @@ class Tree<TreeDataType extends BasicDataNode = DataNode> extends React.Componen
       newState.loadedKeys = props.loadedKeys;
     }
 
+    // ================= dragging ===================
+    if (props.dragging === true) {
+      newState.dragging = props.dragging;
+    }
```

Comment:
If it's possible to use a global variable to save the dragging state rather than let user passing it in?

Reply:
```
let someTreeDragging = false

{
  onDragOver () {
    if (someTreeDragging && !this.state.dragging) { this.setState(dragging) }
  },
  onDragStart() {
    someTreeDragging = true
  }
}

```

Reply:
Would be fine as well

Reply:
Du you want a fixed "magic" variable name, or should we pass the variable name in props as a string?

Reply:
> le to use a global variable to save the dragging state r

Is there a need for magic name? I think we can hold it in a varaible in the package.